appengine/headless-chrome/app.js

```diff
@@ -29,7 +29,7 @@ app.use(async (req, res) => {
 
   // [START browser]
   const browser = await puppeteer.launch({
-    args: ['--no-sandbox', '--disable-setuid-sandbox']
+    args: ['--no-sandbox']
```

Comment:
has this been confirmed in a deplyoment? If I recall the `setuid-sandbox` was necessary on standard

Reply:
Tests are green with and without the flag. @steren, how much confidence do you have in your test? :wink:

Reply:
Ran it manually on standard. Works fine without the flag.